packages/graphiql/package.json

```diff
@@ -43,7 +29,7 @@
     "dev": "babel-node --root-mode upward test/server.js",
     "prepublish": ". ./resources/prepublish.sh",
     "preversion": ". ./resources/checkgit.sh && npm test",
-    "test": "jest --no-cache spec"
+    "test": "node ../../resources/runTests ENZYME=true"
```

Comment:
It would be good if we could eventually move away from enzyme. It's not recommended for React development.

Reply:
lol, I just adopted it, but I'm all about what FB engineers have to say about testing react, haha! so the built-in https://reactjs.org/docs/test-renderer.html instead?

Reply:
There's a nice wrapper around it with react-testing-library (https://github.com/testing-library/react-testing-library). It's fine to have enzyme for now, but it often lags behind React, because it uses unsafe/unstable internal APIs.